packages/react-components/react-dialog/etc/react-dialog.api.md

```diff
@@ -9,9 +9,9 @@
 import { ARIAButtonResultProps } from '@fluentui/react-aria';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 3 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27753&commitId=32f3503a0348ee5af8b7716540e2e83873e64758&env=prod)

✅ There was 0 screenshots added, 3 screenshots removed, 1933 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|0| Removed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|0| Removed|
|DatePicker Compat.default.opened.chromium.png|0| Removed|

</details>


<div id="vrtCommentfluentuiv9"/>